web/app/features/header/Header.tsx

```diff
@@ -4,12 +4,23 @@ import { PostStamp } from '../article/PostStamp'
 
 import { BekkLogo } from '~/features/article/BekkLogo'
 import { useBreadcrumbs } from '~/hooks/useBreadcrumbs'
+import algoliasearch from 'algoliasearch'
+import { Search } from '~/components/Search'
 
-export const Header = () => {
+export const Header = ({
+  algolia,
+}: {
+  algolia?: {
+    app: string
+    key: string
+    index: string
+  }
+}) => {
   const breadcrumbs = useBreadcrumbs()
   const error = useRouteError()
   const { year, date, slug } = useParams()
   const isOnArticlePage = Boolean(year && date && slug)
+  const searchClient = algolia && algoliasearch(algolia.app, algolia.key)
```

Comment:
Jeg mener å huske at man ikke skal lage denne klienten per render. Da får du en feilmelding i consolen.

For å fikse det, så kan du wrappe den i `useRef`, slik:

```suggestion
  const searchClient = useRef(algolia && algoliasearch(algolia.app, algolia.key))
```